homeassistant/components/homewizard/strings.json

```diff
@@ -27,5 +27,145 @@
       "unknown_error": "[%key:common::config_flow::error::unknown%]",
       "reauth_successful": "Enabling API was successful"
     }
+  },
+  "entity": {
+    "button": {
+      "status_light_brightness": {
+        "name": "Status light brightness"
+      }
+    },
+    "sensor": {
+      "dsmr_version": {
+        "name": "DSMR version"
+      },
+      "meter_model": {
+        "name": "Smart meter model"
+      },
+      "unique_meter_id": {
+        "name": "Smart meter identifier"
+      },
+      "wifi_ssid": {
+        "name": "Wi-Fi SSID"
+      },
+      "active_tariff": {
+        "name": "Active tariff"
+      },
+      "wifi_strength": {
+        "name": "Wi-Fi strength"
+      },
+      "total_power_import_kwh": {
+        "name": "Total power import"
+      },
+      "total_power_import_t1_kwh": {
+        "name": "Total power import T1"
+      },
+      "total_power_import_t2_kwh": {
+        "name": "Total power import T2"
+      },
+      "total_power_import_t3_kwh": {
+        "name": "Total power import T3"
+      },
+      "total_power_import_t4_kwh": {
+        "name": "Total power import T4"
+      },
+      "total_power_export_kwh": {
+        "name": "Total power export"
+      },
+      "total_power_export_t1_kwh": {
+        "name": "Total power export T1"
+      },
+      "total_power_export_t2_kwh": {
+        "name": "Total power export T2"
+      },
+      "total_power_export_t3_kwh": {
+        "name": "Total power export T3"
+      },
+      "total_power_export_t4_kwh": {
+        "name": "Total power export T4"
+      },
+      "active_power_w": {
+        "name": "Active power"
+      },
+      "active_power_l1_w": {
+        "name": "Active power L1"
```

Comment:
For L1, L2 and L3, maybe we should explicitly call them 'phase 1', 'phase 2' and 'phase 3' to prevent confusion during/after translation?